bedrock/settings/base.py

```diff
@@ -453,10 +453,15 @@ def JINJA_CONFIG():
 # This can be changed to use session once we do add a database.
 MESSAGE_STORAGE = 'django.contrib.messages.storage.cookie.CookieStorage'
 
-EMAIL_BACKEND = config(
-    'EMAIL_BACKEND',
-    default='django.core.mail.backends.console.EmailBackend' if DEBUG else
+
+def lazy_email_backend():
```

Comment:
I think this is a fine solution, but the other way we solve these kinds of things is to have them in `settings/__init__.py`. I'm thinking we'll eventually want to get everything out of `__init__.py`, so my question is whether you think this PR is a good chance to do some of that?